files/.prettierignore

```diff
@@ -1,19 +1,10 @@
 # Prettier is also run from each package, so the ignores here
 # protect against files that may not be within a package
 
-# dependencies
-/node_modules/
-
 # misc
 !.*
-.eslintcache
 .lint-todo/
 
 # ember-try
 /.node_modules.ember-try/
-/bower.json.ember-try
```

Comment:
The `.ember-try` extension is not something Prettier recognises, so these files don't get picked up by default as well.